src/Elastic.Apm/Helpers/DbConnectionStringParser.cs

```diff
@@ -19,7 +19,12 @@ internal class DbConnectionStringParser
 		// (for example see https://github.com/dotnet/corefx/issues/3357)
 		private volatile int _cacheCount;
 
-		internal DbConnectionStringParser(IApmLogger logger) => _logger = logger.Scoped(ThisClassName);
+		internal const int MaxNestingDepth = 100;
+
+		internal DbConnectionStringParser(IApmLogger logger)
+		{
+			_logger = logger.Scoped(ThisClassName);
+		}
```

Comment:
nit: expression bodied members 
<img width="435" alt="image" src="https://user-images.githubusercontent.com/1091853/78165029-0399c600-744b-11ea-8731-fade0e782964.png">
```suggestion
		internal DbConnectionStringParser(IApmLogger logger) => _logger = logger.Scoped(ThisClassName);
```